### DISCUSSION TITLE
Is there a way that in form that when user input records it will automatically uppercase even in front end?

### DISCUSSION THREAD

**Discussion by christopherlllobrera** [filamentphp/filament#13246] [General]
I searched all the documentation the only option is autocapitalize.  Which is  capitalize the first letter of the word I input.

**Comment by christopherlllobrera** [comment]
I found it

->extraInputAttributes(['onChange' => 'this.value = this.value.toUpperCase()'])

**Reply by MrkDchvz** [reply]
Thanks!

**Reply by Ybansan** [reply]
Even better:

->extraInputAttributes(['onInput' => 'this.value = this.value.toUpperCase()'])

**Reply by tedevs0** [reply]
Thanks!
